contracts/samples/VerifyingPaymaster.sol

```diff
@@ -50,8 +50,8 @@ contract VerifyingPaymaster is BasePaymaster {
                     userOp.nonce,
                     keccak256(userOp.initCode),
                     keccak256(userOp.callData),
-                    userOp.callGasLimit,
-                    userOp.verificationGasLimit,
+                    userOp.accountGasLimits,
+                    uint128(bytes16(userOp.paymasterAndData[20:52])),
```

Comment:
afaik, VerifyingPaymaster should verify both pmgl and postgl, use entire 256 bits.

Reply:
Fixed